__tests__/server.test.js

```diff
@@ -0,0 +1,42 @@
+const app = require("../app");
+const request = require("supertest");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const { topicData, userData, articleData, commentData } = require("../db/data/test-data/index");
+
+beforeEach(() => seed({ topicData, userData, articleData, commentData }));
+afterAll(() => db.end());
+
+describe("incorrect path tests", () => {
```

Comment:
While these tests are most likely passing, the way you've handled them is not correct.